setup.cfg

```diff
@@ -30,7 +30,7 @@ include_package_data = True
 install_requires =
     napari >=0.4.13
     scikit-image
-    pymmcore-plus >=0.3.1
+    pymmcore-plus >=0.4.0
```

Comment:
this is presuming we bump the version numbers that much - which I think makes sense?

Reply:
for the purpose of this PR, let's set this to the main branch 
something like
```
install_requires =
  pymmcore-plus  @ git+https://github.com/tlambert03/pymmcore-plus
```

Reply:
then once everything is settled, we can cut a new release and actually depend on 0.4.